docs/analytics.rst

```diff
@@ -0,0 +1,80 @@
+.. This Source Code Form is subject to the terms of the Mozilla Public
+.. License, v. 2.0. If a copy of the MPL was not distributed with this
+.. file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+============================
+Analytics
+============================
+
+Google Tag Manager (GTM)
+------------------------
+
+Bedrock uses Google Tag Manager (GTM) to manager and organize it's Google Analytics solution. GTM is a tag management system that allows for easy implementation of Google Analytics tags and other 3rd party marketing tags in a nice GUI experience. Tags can be added, updated, or removed directly from the GUI. Bedrock's GTM solution is CSP compliant and does not allow for the injection of custom HTML or JavaScript but all tags use built in templates so minimize any chance of introducing a bug into Bedrock. GTM allows for a `one source of truth` approach to managing an analytics solution in that all analytics tracking can be inside GTM.
+
+The GTM DataLayer
+-----------------
+
+How an application communicates with GTM is via the dataLayer object, which is a simple JavaScript array GTM instantiates on the page. Bedrock will send messages to the dataLayer object by means of pushing an object literal onto the dataLayer. GTM creates an abstract data model from these pushed objects that consists of the most recent value for all keys that have been pushed to the dataLayer. The only reserved key in an object pushed to the dataLayer is ``event`` which will cause GTM to evaluate the firing conditions for all tag triggers.
```

Comment:
``dataLayer`` should probably be styled as a piece of code? (same as you have done for ``event`` below)